pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,112 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, stdenv
+, fetchzip
+, greenlet
+, websockets
+, pyee
+, pillow
+, pixelmatch
+, autobahn
+, pyopenssl
+, service-identity
+, requests
+, objgraph
+}:
+
+buildPythonPackage rec {
+  pname = "playwright";
+  version = "1.21.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "playwright-python";
+    rev = "v${version}";
+    sha256 = "sha256-+q95w1UMCEZeYeXvlLagixUAROakdJFrIBy1rPMyYZc=";
+  };
+
+  format = "pyproject";
+
+  playwright-driver = stdenv.mkDerivation {
```

Comment:
move this into another file, `./driver.nix` or something

Reply:
I had it separate initially, however because they both rely on the version being the same, I thought it made more sense to keep them in same file so I could "inherit" the version.

Reply:
`driver.nix`
```nix
{ fetchzip, whatever, elsee, playwrightVersion }:

stdenv.mkDerivation { ... }
```
`default.nix`
```nix
  playwright-driver = callPackage ./driver.nix { playwrightVersion = version; }
  # maybe pkgs.callPackage, I dunno
```

Reply:
ah, that's what I was missing. Thanks so much <3